app/config/config.yml

```diff
@@ -35,6 +35,7 @@ twig:
     globals:
         role_voir_confidentiel: "%role_voir_confidentiel%"
         authorizedStorageSize: "%junior_authorizedstoragesize%"
+        technical_to: "%technical_to%"
```

Comment:
No need to  create a global variable. `{{ app.environnment.technical_to }}` should work